tests/qflags/grep-results/qt3d-qflag-grep-result.txt

```diff
@@ -0,0 +1,8 @@
+src\animation\frontend\qanimationcallback.h:    Q_DECLARE_FLAGS(Flags, Flag)
```

Comment:
These are supposed to be committed?

Reply:
Yes, this gives the reproducibility. Without this, it looks a bit like black magic to identify what is a potential QFlag. With it, it is possible to run (manually) the full process and reach the same result.